src/parquet/CMakeLists.txt

```diff
@@ -25,3 +25,4 @@ install(FILES
   DESTINATION include/parquet)
 
 ADD_PARQUET_TEST(reader-test)
+target_link_libraries(reader-test ${LINK_LIBS} -lz)
```

Comment:
Can you please add this to the linked libraries for `libparquet_compression.a` in `src/parquet/compression/CMakeLists.txt`?